tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,59 @@
+"""Work with inventory."""
+
+
+def display_inventory(inventory):
+    """Print inventory."""
+    print('Inventory:')
+    items = 0
+    # check that inventory is a dict
+    if not isinstance(inventory, dict):
```

Comment:
Usually we don't do such checks in Python. If you want some more type checking you may take a look at [type hints in Python](https://www.python.org/dev/peps/pep-0484/#abstract). It does **not** perform check like your code here, but it should integrate smoothly with your editor. Depending on editor configuration you will get at least better suggestions and possibly warnings in case of mismatched types. Also, at some point in the future you may want to use static type checker, which makes use of type hints.

Of course it is totally optional :)

Reply:
I'd just add that if you'd like to stick with this check, then exit conditions should be as high in the function as possible, in this case, you could move it to line 6 :)